curator/patterns/post-comments.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Title: Post Comments
+ * Slug: curator/post-comments
+ * Inserter: no
+ */
+
+?>
+
+<!-- wp:group {"layout":{"inherit":true}} -->
+<div class="wp-block-group"><!-- wp:post-comments-form /--></div>
+<!-- /wp:group -->
+
+<!-- wp:spacer {"height":"60px"} -->
+<div style="height:60px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
```

Comment:
Adjust the spacing

<img width="337" alt="image" src="https://user-images.githubusercontent.com/1935113/169808175-68ee6e2e-2cd5-42ca-ae43-b301fb81e194.png"> <img width="339" alt="image" src="https://user-images.githubusercontent.com/1935113/169808275-b6653f65-422b-4928-932d-3cb5ca5d7dd4.png">